web/package.json

```diff
@@ -31,99 +31,99 @@
         "**/*.{js,cjs,mjs,jsx,ts,cts,mts,tsx}": "eslint --fix"
     },
     "devDependencies": {
-        "@emotion/jest": "11.10.5",
-        "@graphql-codegen/cli": "2.16.3",
-        "@graphql-codegen/fragment-matcher": "3.3.3",
-        "@graphql-codegen/introspection": "2.2.3",
-        "@graphql-codegen/typescript": "2.8.7",
-        "@graphql-codegen/typescript-operations": "2.5.12",
+        "@emotion/jest": "11.11.0",
+        "@graphql-codegen/cli": "3.3.1",
+        "@graphql-codegen/fragment-matcher": "4.0.1",
+        "@graphql-codegen/introspection": "3.0.1",
+        "@graphql-codegen/typescript": "3.0.4",
+        "@graphql-codegen/typescript-operations": "3.0.4",
         "@graphql-codegen/typescript-react-apollo": "3.3.7",
-        "@playwright/test": "1.29.2",
-        "@rollup/plugin-yaml": "4.0.1",
-        "@storybook/addon-essentials": "6.5.15",
-        "@storybook/addons": "6.5.15",
-        "@storybook/builder-vite": "0.2.7",
-        "@storybook/react": "6.5.15",
-        "@storybook/theming": "6.5.15",
+        "@playwright/test": "1.33.0",
+        "@rollup/plugin-yaml": "4.1.0",
+        "@storybook/addon-essentials": "7.0.11",
+        "@storybook/addons": "7.0.11",
+        "@storybook/builder-vite": "7.0.11",
+        "@storybook/react": "7.0.11",
+        "@storybook/theming": "7.0.11",
         "@testing-library/jest-dom": "5.16.5",
-        "@testing-library/react": "13.4.0",
+        "@testing-library/react": "^14.0.0",
```

Comment:
Please remove all ^ and then regen yarn.lock

Reply:
done